Components/webComponents/cartComponents/CartItem.js

```diff
@@ -38,27 +38,44 @@ export class CartItem extends HTMLElement {
         this.querySelector(".cart-delete-item").addEventListener(
             "click",
             function () {
-                publishEvent("removeItem", this.productId);
+                this.dispatchEvent(
+                  new CustomEvent("onItemRemoved", {
+                      detail: this.productId,
+                      bubbles: true
```

Comment:
@Komplett-Jamie Why is it important that we set `bubbles: true`  in events that are dispatched from `CartItem`?

Reply:
bubbles true makes it possible for things to be passed through to the parents and up through the DOM.